charts/access/slack/values.yaml

```diff
@@ -0,0 +1,64 @@
+# Default values for slack.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# replicaCount: 1
+
+image:
+  repository: 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-plugin-slack
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+resources: {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+#
+# Plugin specific options
+#
+teleport:
+  address: ""
+  identitySecretName: ""
+  identitySecretPath: "auth_id"
+
+slack:
+  token: ""
+
+roleToRecipients:
+  "*":
+    - "recipient1@example.com"
+    - "recipient2@example.com"
+  dev:
+    - "another-recipient@example.com"
```

Comment:
It'd be good to be consistent with what you put in README.md and values.schema.json.  I also like providing an example of both a channel for approvals and users for approvals.  For reference:

```
  <tr>
    <td><code>roleToRecipients</code></td>
    <td>
      Mapping of roles to a list of channels and Slack emails. <br />
      Example:
      <pre>
"dev" = ["dev-access-requests", "user@example.com"]
"*" = ["access-requests"]</pre>
    </td>
    <td>map</td>
    <td><code>{}</code></td>
    <td>yes</td>
  </tr>
```

Reply:
Actually I've realized here that those should not be there as defaults. I put `{}` there as a solution